app/services/permissions_service.rb

```diff
@@ -43,6 +46,7 @@ def guest_permissions
       return true if controller == "skills" && action == "show"
       return true if controller == "professionals" && action == "index"
       return true if controller == "sessions" && action.in?(%w(new create))
+      # a registered user should not be authorized for line 50
       return true if controller == "users" && action.in?(%w(new create))
```

Comment:
Yeah I agree with that logic. But I also think it's a great idea to all have a discussion about this @rtravitz! @AliSchlereth @alishersadikov